content/blog/2016/2016-05-04-the-state-of-JAM

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title: "The State of Jenkins Area Meetup (JAM)"
+tags:
+- meetup
+- JAM
+- event
+author: alyssat
+---
+
+NOTE: _This is a guest post by link:https://github.com/alyssat[Alyssa Tong].
+Alyssa works for link:https://www.cloudbees.com[CloudBees], helping to organize
+Jenkins community events link:https://www.meetup.com/pro/jenkins[around the
+world]._
+
+Recently, the Jenkins project announced the release of link:https://jenkins.io/blog/2016/04/26/jenkins-20-is-here/[Jenkins 2.0], a first major release after 10 years and 655 weekly releases. This has been a major milestone for Jenkins and its growing community of developers, testers, designers and other users in the software delivery process. 
+
+With its rising popularity and wide adoption, the Jenkins community continues to grow and evolve into the millions. Jenkins community meetup activity has risen to an all time high since the first Jenkins meetup which was established on August 23 2010, in San Francisco. 
+
+Over the last six months the link:http://www.meetup.com/pro/jenkins/[Jenkins Area Meetup (JAM) Groups] has grown from 5 to 30 groups with coverage in Asia, North America, South America and Europe - collectively.  That’s an average growth of 4 new JAM groups per month.
+
+image:https://www.cloudbees.com/sites/default/files/jam-map.png[JAM map,500,float="right"]
+
+As of today, there are over *4,100* Jenkins fans within the Jenkins meetup community.  This is the result of contributions from community JAM leaders who have volunteered their time to provide a platform for learning, sharing and networking all things Jenkins within their local communities. 
+
+For anyone who has not organized a meetup before, there are many moving parts that have to come together at a specific location, date and time. This process takes months to methodically plan out - from planning the food and beverages to securing speaker(s), venue, audio/visual setup, technical logistics to promoting the meetup. It does take a level of passion and effort to make it all happen.  
+
+Many *THANKS* to the *55* JAM leaders, who encompass this passion - they have successfully organized over *41* meetups within the past six months specifically in North America, South America and Europe. That’s about 6-7 meetups a month.
+
+image:https://www.cloudbees.com/sites/default/files/jams-over-time.png[JAMs over time,500,float="right"]
+
+Here are a few images of JAM activities ...
+
+image:https://www.cloudbees.com/sites/default/files/toulouse-jam.png[Toulouse JAM 1,500,float="right"]
+
+image:https://www.cloudbees.com/sites/default/files/toulouse-jam-2.png[Toulouse JAM 2,500,float="right"]
```

Comment:
I agree, I don't have the permissions to upload images to the Jenkins 'images' folder, so I'm hoping that adding the images in like this will show that I do want images in there, I just don't know how to add correctly.